examples/complete-vpc/main.tf

```diff
@@ -1,20 +1,87 @@
 provider "aws" {
+  region = local.region
+}
+
+locals {
+  name   = "complete-example"
   region = "eu-west-1"
+  tags = {
+    Owner       = "user"
+    Environment = "staging"
+    Name        = "complete"
+  }
 }
 
+################################################################################
```

Comment:
Please move all "supporting resources" somewhere below the main `module "vpc"` block

Reply:
updated in https://github.com/terraform-aws-modules/terraform-aws-vpc/pull/635/commits/5c6a2313e49d8f537b8d0e4f45f827c62d742541